webapp/src/components/common/Matrix/components/MatrixFilter/hooks/useTemporalData.ts

```diff
@@ -15,17 +15,17 @@
 import { useMemo } from "react";
 import type { TimeFrequencyType } from "../../../shared/types";
 import { TIME_INDEXING } from "../constants";
-import { extractValueFromDate, getDefaultRangeForIndexType } from "../utils/dateUtils";
-
+import { getDefaultRangeForIndexType } from "../utils/dateUtils";
+import type { DateInfo } from "@/components/common/Matrix/components/MatrixFilter/types";
+import { INDEX_TYPE_TO_DATEINFO_PROPERTY } from "@/components/common/Matrix/components/MatrixFilter/utils";
 interface UseTemporalDataProps {
-  dateTime?: string[];
+  datesInfo?: DateInfo[];
   isTimeSeries: boolean;
   timeFrequency?: TimeFrequencyType;
 }
```

Comment:
since we no longer use `isTimeSeries` and `timeFrequency` we can remove them from the interface, or even remove it and use directly `export function useTemporalData(datesInfo?: DateInfo[]) {`